### PR TITLE
robotic.mobi: adding options 'source' capability

### PR DESCRIPTION
**Pull request by donmezburak**
robotic.mobi: adding options 'source' capability

### CODE REVIEW

**Comment by donmezburak** [comment]
Done. When will this commit merge into master and publish as product ?

**Comment by donmezburak** [comment]
Sorry I see the problem now. Fixed it.

File: app/renderer/actions/Session.js [hunk 1]
```diff
@@ -386,8 +386,9 @@ export function newSession (caps, attachSessId = null) {
         port = 443;
         https = session.server.roboticmobi.ssl = true;
         if (caps) {
-          desiredCapabilities.robotic_mobi_token = session.server.roboticmobi.token || process.env.ROBOTIC_MOBI_TOKEN;
-        }
+          desiredCapabilities['roboticmobi:options'] = {};
+          desiredCapabilities['roboticmobi:options'].robotic_mobi_token = session.server.roboticmobi.token || process.env.ROBOTIC_MOBI_TOKEN;
+        }
```
[the remark this answers is not in the document]
Comment (donmezburak) [reply app/renderer/actions/Session.js]: Done

**Review by KazuCocoa — APPROVED** [review]

**Comment by jlipps** [comment]
Hmm I just noticed this was merged into `jlipps/fix-package` instead of `master`. It should probably be in `master` instead?

**Comment by jlipps** [comment]
I mean `main`

**Comment by KazuCocoa** [comment]
oh, yea..

**Comment by donmezburak** [comment]
I thought you will merge into main. If you wish, I can create another pr for main

**Comment by jlipps** [comment]
I have merged it into main. FWIW, the branch you targeted was for an unrelated change, that was partly why I was confused.

**Comment by donmezburak** [comment]
When I saw your mail , I thought this branch I would have to use to create pr. Thanks for reply and merge.

Screen Shot 2021-09-23 at 11 23 42

**Pull request merged**